portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+var $profile = $('#profile'),
+    lFollowX = 0,
+    lFollowY = 0,
+    lFollowZ = 0,
+    x = 0,
+    y = 0,
+    friction = 1 / 10;
+
+$(document).ready(function() {
+  function animate() {
```

Comment:
OOC: Why does this method need to be defined when the document is ready? Why can't be a static function?

Reply:
My reasoning behind using the $(document).ready is centered around simplifying what the parallax effect affects. By using document.ready, then as soon as the DOM is ready to execute JS, then the code will always execute anytime the appropriate action is taken. Also, this gives greater flexibility for which elements can be given a parallax effect. I also, over the course of development, attempted to attach this directly to some mouse event, but eventually found this method to be a lot more reliable.